SDKBuildScripts/unity_RunAutoTests.sh

```diff
@@ -201,7 +201,7 @@ TryBuildAndTestiOS() {
                 if [[ $? -ne 0 ]]; then return 1; fi
 
                 #run the downloaded IPA on AppCenter Test
-                . ./runAppCenterTest.sh "$WORKSPACE/SDKGenerator/SDKBuildScripts/PlayFabIOS.ipa" "$WORKSPACE/SDKGenerator/SDKBuildScripts/AppCenterUITestLauncher/AppCenterUITestLauncher/debugassemblies" ios || exit 1
+                ./runAppCenterTest.sh "$WORKSPACE/SDKGenerator/SDKBuildScripts/PlayFabIOS.ipa" "$WORKSPACE/SDKGenerator/SDKBuildScripts/AppCenterUITestLauncher/AppCenterUITestLauncher/debugassemblies" unity-ios || exit 1
```

Comment:
>. [](start = 16, length = 1)

We have double-dots above, for unity_*** scripts. We probably want to make it consistent: same here and there

Reply:
the start of the line:
. ./runAppCenterTest.sh
is required for now. I figured out why it's required, and that's technically a bug, but it'll randomly fail until I have the time to fix it across the board.
Keep the . ./etc for now.